packages/components/src/entities/SampleTypeAppContext.tsx

```diff
@@ -1,13 +1,44 @@
+import { Filter } from '@labkey/api';
+
 import { ExtendableAppContext, useAppContext } from '../internal/AppContext';
 import { DomainDetails } from '../internal/components/domainproperties/models';
+import { User } from '../internal/components/base/models/User';
+import { SchemaQuery } from '../public/SchemaQuery';
+import { ALIQUOT_FILTER_MODE } from '../internal/components/samples/constants';
+import { QueryConfigMap } from '../public/QueryModel/withQueryModels';
+
+import { JobsButton, SampleStorageButton, WorkflowGrid } from '../internal/components/samples/models';
+import { DetailRenderer } from '../internal/components/forms/detail/DetailDisplay';
+
+import { ReferencingNotebooks, SampleStorageMenu, SampleStorageLocation } from './models';
 
 export interface SampleTypeAppContext {
+    JobsButtonComponent: JobsButton;
```

Comment:
I agree. I wasn't a fan of putting the premium components into the context but I couldn't come up with another way to get the sharing of the sample page components that we wanted AND having a way to use premium components in the app on these pages. I'm not following how this will change once we introduce a premium package. Won't we still have scenarios where a shared ui-components component needs to render a premium component but we can't / don't want to make it a circular package dependency? Even if we take this /entities subpackage out into the premium package, we still don't want dependencies that go from workflow -> entities -> workflow. Maybe I'm missing something.